### ISSUE TITLE
Beta is not a new release (it is a preview) 

### ISSUE THREAD

**Issue by sdtbluethink** [archon810/apkmirror-public#38]
It would be great to split apps in release and beta.
For example I'm not interested in Facebook lite beta, but the latest download is everytime the beta release. 

The other point is the RSS-Feed of this app. It is not possible to subscribe release or beta only. It is needed to subscribe booth in the general feed.

What's your option?

**Comment by archon810** [comment]
@SeBsZ What do you think?

**Comment by SeBsZ** [comment]
I agree we could maybe add a way to filter these lists to just stable, alpha, beta etc.
